backend/pdf.go

```diff
@@ -183,19 +218,20 @@ func GeneratePDFTicket(event *zeni.Event, ticketSecret string, logger *zap.Logge
 	return buf.Bytes(), nil
 }
 
-func drawImagePlaceholder(pdf *fpdf.Fpdf, x, y, size float64) {
+func drawImagePlaceholder(pdf *fpdf.Fpdf, x, y, width, height float64) {
 	pdf.SetDrawColor(180, 180, 180)
 	pdf.SetFillColor(240, 240, 240)
-	pdf.Rect(x, y, size, size, "FD")
+	pdf.Rect(x, y, width, height, "FD")
 
 	pdf.SetFont("Helvetica", "I", 12)
 	pdf.SetTextColor(120, 120, 120)
-	pdf.SetXY(x, y+size/2-5)
-	pdf.Cell(size, 10, "Event Image")
+	pdf.SetXY(width/2-5, y+height/2-5)
+	pdf.Cell(0, 10, "Event Image")
 }
 
 func embedImageURL(pdf *fpdf.Fpdf, imageURI string, x, y, width, height float64) error {
-	resp, err := http.Get(imageURI)
+	optimizedURL := fmt.Sprintf("%s?img-width=1240&img-height=698&img-quality=75&img-fit=crop", imageURI)
```

Comment:
```suggestion
	optimizedURL := fmt.Sprintf("%s?img-width=1240&img-height=698&img-quality=75&img-fit=cover", imageURI)
```
I think this would be better so we get the maximum content we can get while respecting aspect ratio
This is what is used in frontend I believe

Reply:
Indeed, i confirm after few tests that cover answer better our needs [7bcd686](https://github.com/samouraiworld/zenao/pull/472/commits/7bcd686258d6ccbcd42fcd9b727d7904052ee071)